<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>189</red>
        <green>189</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>189</red>
        <green>189</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>189</red>
        <green>189</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>189</red>
        <green>189</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Graft 0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="actionsVLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Actions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="operationsListWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionlistButtonsLayout">
        <item>
         <widget class="QPushButton" name="addActionButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add operation</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeActionButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove selected operation</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="addActionComboBox">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>498</width>
            <height>26</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Type of operation to add (temp ui)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="InspectorWidget" name="actionInspector">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>429</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="outputLayout">
      <item>
       <layout class="QGridLayout" name="inputRenderLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outputButtonsLayoyt">
        <item>
         <widget class="QPushButton" name="exportButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="processSelectedOnlyCheckbox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string extracomment="Only process the currently selected file"/>
          </property>
          <property name="text">
           <string>Selected </string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="runConvertorButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="InspectorWidget" name="outputInspector">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>100</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>448</width>
           <height>98</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="filesLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inputs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="inputListWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filesListButtonsLayout">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item>
         <widget class="QPushButton" name="addInputFilesButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add file or folder</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeInputFilesButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove selected file</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="InspectorWidget" name="fileInspector">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>269</width>
           <height>429</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionSave_Actions_Queue"/>
    <addaction name="actionLoad_Actions_Queue"/>
   </widget>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave_Actions_Queue">
   <property name="text">
    <string>Save Actions Queue</string>
   </property>
  </action>
  <action name="actionLoad_Actions_Queue">
   <property name="text">
    <string>Load Actions Queue</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InspectorWidget</class>
   <extends>QScrollArea</extends>
   <header>InspectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionSave_Actions_Queue</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_project_saveActionQueue_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_Actions_Queue</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_project_loadActionQueue_triggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_project_saveActionQueue_triggered(bool)</slot>
  <slot>on_project_loadActionQueue_triggered(bool)</slot>
 </slots>
</ui>
